app/views/works/_work_module.html.erb

```diff
@@ -99,12 +99,12 @@
 
   <% if work.count_visible_comments > 0 %>
     <dt class="comments"><%= ts('Comments') %>:</dt>
-    <dd class="comments"><%= link_to work.count_visible_comments, work.chapter_total_display.to_i > 1 ? work_path(work, anchor: "comments", show_comments: true, view_full_work: "true") : work_path(work, anchor: "comments", show_comments: true) %></dd>
+    <dd class="comments"><%= link_to work.count_visible_comments, work.number_of_posted_chapters > 1 ? work_path(work, anchor: "comments", show_comments: true, view_full_work: "true") : work_path(work, anchor: "comments", show_comments: true) %></dd>
```

Comment:
Oh, you know what, I think there's a cache key on line 94 that needs to be bumped from v2 to v3. Sorry for not noticing that before!

Reply:
The resulting HTML should not be affected by this change, it's just about how it gets there. Does the cache key still have to be updated then?

For clarity, the blurb is not affected be the delimiters in the chapter counts because it goes through chapter_total_display_with_link which is not changed here but in #4319 (and that would also be a different cache key, line 27).

Reply:
I just realized that the chapter display in work blurbs _is_ affected, in a specific case: One posted chapter, some amount of planned chapters. E.g. 1/2000 chapters should now be displayed as to 1/2,000 chapters. But something like 300/5000 will stay the same. Sorry about the confusion!

So I updated both keys, the one in line 94 for this change and the one in line 27 for the chapter count display change.

Reply:
All good, and thanks!